tests/mongo.yml

```diff
@@ -1,22 +1,89 @@
 ---
 runner: jstest
 args:
+  - mongo/jstests/core/all*.js
+  - mongo/jstests/core/array[1-4].js
+  - mongo/jstests/core/arrayfind[1-9].js
   - mongo/jstests/core/basic*.js
-  - mongo/jstests/auth/auth1.js
```

Comment:
I'm missing the context on why we are removing this `mongo/jstests/auth/auth1.js` :thinking:

Reply:
Fixed